### PR TITLE
Extend `Account.create()` method with `user_package` parameter

### PR DESCRIPTION
**Pull request by Toshakins**
Relates to https://github.com/yola/sitewitservice/issues/288

### CODE REVIEW

File: sitewit/models.py [hunk 1]
```diff
@@ -41,8 +41,8 @@ def __init__(self, account_data, user_data=None):
             self.user = None
 
     @classmethod
-    def create(
-            cls, user, url, site_id=None, user_token=None, mobile_phone=None):
+    def create(cls, user, url, site_id=None, user_token=None,
```
Comment (snitch) [inline sitewit/models.py]: D413 Missing blank line after last section

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: sitewit/models.py [hunk 2]
```diff
@@ -52,6 +59,8 @@ def create(
             user_token (str, optional): user token. Is specified if the user
                 has another accounts.
             mobile_phone (str, optional): account owner's phone
+            user_package (str, optional): package subscription of account's
+                                            owner
```
Comment (a-milogradov) [inline sitewit/models.py]: inconsistent indentation spotted 🤓

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**